crates/iota-indexer/src/store/pg_indexer_store.rs

```diff
@@ -2222,38 +2183,87 @@ fn make_objects_history_to_commit(
     deleted_objects.into_iter().chain(mutated_objects).collect()
 }
 
-fn make_objects_snapshot_to_commit(
+// Partition object changes into deletions and mutations,
+// within partition of mutations or deletions, retain the latest with highest
+// version; For overlappings of mutations and deletions, only keep one with
+// higher version. This is necessary b/c after this step, DB commit will be done
+// in parallel and not in order.
+fn retain_latest_indexed_objects(
```

Comment:
I'm wondering if this could be converted into a function doc using `///`, the info seems quite important even tough is a private function.